### PR TITLE
Setup cms

### PR DESCRIPTION
**Pull request by GrandSchtroumpf**
Setup CMS

### CODE REVIEW

File: styles/typography.scss [hunk 1]
```diff
@@ -19,5 +29,7 @@ $bf-typography: mat-typography-config(
 
 pre {
   font-family: mat-font-family($bf-typography);
-  font-size: mat-font-size($bf-typography, body-1);
-}
\ No newline at end of file
+  font-size: mat-font-size($bf-typography, body-1); // Get size of body-1
+}
+
+// TODO : Create a builder for typography with two polices
```
Comment (phpgeekfr) [inline styles/typography.scss:35]: clean comments if not needed  and maybe add an issue number to TODOs

File: libs/utils/src/lib/routes/create-routes.ts [hunk 2]
```diff
@@ -17,6 +17,9 @@ interface RouteOptions {
 }
 
 export function createRoutes({ appsRoutes, appName, landing }: RouteOptions) {
+  // Used for internal app
+  landing = landing || { path: '', redirectTo: 'c', pathMatch: 'full' };
```
Comment (phpgeekfr) [inline libs/utils/src/lib/routes/create-routes.ts:21]: is it ok to reassign a var comming from parameters?

File: apps/cms/src/environments/environment.prod.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const environment = {
+  production: true,
+};
```
Comment (phpgeekfr) [inline apps/cms/src/environments/environment.prod.ts:3]: I don't think other apps have there own env file. Is this needed?

File: apps/cms/src/environments/environment.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+};
+
+/*
+ * For easier debugging in development mode, you can import the following file
+ * to ignore zone related error stack frames such as `zone.run`, `zoneDelegate.invokeTask`.
+ *
+ * This import should be commented out in production mode because it will have a negative impact
+ * on performance if an error is thrown.
+ */
+// import 'zone.js/dist/zone-error';  // Included with Angular CLI.
```
Comment (phpgeekfr) [inline apps/cms/src/environments/environment.ts:16]: I don't think other apps have there own env file. Is this needed?

File: apps/cms/src/app/shell/shell.module.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import { RouterModule, Route } from '@angular/router';
+import { ShellComponent } from './shell.component';
+
+const children: Route[] = [{
+  path: '',
+  redirectTo: 'festival',
+  pathMatch: 'full',
+}, {
+  path: 'festival',
+  loadChildren: () => import('../festival/festival.module').then(m => m.FestivalModule)
```
Comment (phpgeekfr) [inline apps/cms/src/app/shell/shell.module.ts:12]: you are loading a festival module here

File: apps/cms/src/app/shell/shell.module.ts [hunk 6]
```diff
@@ -0,0 +1,22 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule, Route } from '@angular/router';
+import { ShellComponent } from './shell.component';
+
+const children: Route[] = [{
+  path: '',
+  redirectTo: 'festival',
```
Comment (phpgeekfr) [inline apps/cms/src/app/shell/shell.module.ts:8]: redirectTo: 'festival',

File: apps/cms/src/app/festival/festival.module.ts [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+@NgModule({
+  declarations: [FestivalComponent],
+  imports: [
+    CommonModule,
+    MatSidenavModule,
+    MatListModule,
+    RouterModule.forChild([{ path: '', component: FestivalComponent }])
+  ]
+})
+export class FestivalModule {}
```
Comment (phpgeekfr) [inline apps/cms/src/app/festival/festival.module.ts:19]: file is in cms/src/app/festival ? 
file is named festival.module

File: apps/cms/src/app/festival/festival.component.ts [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class FestivalComponent implements OnInit {
+  pages = pages;
+  constructor() { }
+
+  ngOnInit(): void {
+  }
+
+}
```
Comment (phpgeekfr) [inline apps/cms/src/app/festival/festival.component.ts:21]: same as below

File: apps/cms/src/app/festival/festival.component.html [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+        <a mat-list-item [routerLink]="page.path" routerLinkActive="active">
+          {{ page.label }}
+        </a>
+      </ng-container>
+    </mat-nav-list>
+  </mat-sidenav>
+  <mat-sidenav-content>
+
+  </mat-sidenav-content>
+</mat-sidenav-container>
```
Comment (phpgeekfr) [inline apps/cms/src/app/festival/festival.component.html:14]: same as below

File: apps/cms/src/app/festival/festival.component.scss [hunk 10]
```diff
@@ -0,0 +1,12 @@
...
+  height: 100%;
+}
+
+mat-sidenav-container {
+  height: 100%;
+}
+
+mat-sidenav {
+  width: 250px;
+}
```
Comment (phpgeekfr) [inline apps/cms/src/app/festival/festival.component.scss:12]: same as below

**Review by phpgeekfr — APPROVED** [review]
LGTM

File: libs/utils/src/lib/routes/create-routes.ts (hunk 2 above)
Comment (GrandSchtroumpf) [reply libs/utils/src/lib/routes/create-routes.ts:21]: yes

File: apps/cms/src/app/shell/shell.module.ts (hunk 5 above)
Comment (GrandSchtroumpf) [reply apps/cms/src/app/shell/shell.module.ts:12]: It's the cms for the festival app.

**Pull request merged**